test/repositories/RandomSelectionSpec.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import org.scalatest._
+import org.scalatestplus.play.PlaySpec
+
+class RandomSelectionSpec extends PlaySpec {
+  "RandomSelection" should {
+    "calculate the correct batch size and random offset" in {
+
+      RandomSelection.calculateBatchSize(1, 1) mustBe ((0,1))
+      val (offset, size) = RandomSelection.calculateBatchSize(1, 9)
+      offset must (be >= (0) and be <= (8))
+      size mustBe 1
+
+      1 to 200 foreach { _ =>
+        val numberOfDocs = scala.util.Random.nextInt(10000)
```

Comment:
Can we adjust this value to 1000? It should speed the test up dramatically, and I am not convinced that it lowers our confidence in the validity of the test